docs/using-mycroft-ai/customizations/wake-word.md

```diff
@@ -167,11 +167,17 @@ mycroft-config reload
 
 ## Switching Wake Word Listeners
 
-To change the Wake Word Listener to PocketSphinx if it has been set to Precise, Speak:
+As per the [Technology Overview](https://mycroft-ai.gitbook.io/docs/mycroft-technologies/overview#wake-word-detection), there are two available listeners:
```

Comment:
just a minor refresh/intro (in case the reader hasn't read the technology overview)